src/lib.rs

```diff
@@ -43,6 +47,66 @@ pub enum RequestCode {
     Unknown = u32::MAX,
 }
 
+/// CPU, memory, and disk usage.
+#[derive(Debug, Deserialize, Serialize)]
+pub struct ResourceUsage {
```

Comment:
Any reason to have this in lib.rs? Since it is a type used in request/response, the `request` module is a better place for this.

Reply:
I've gotten it back where it belonged to, `request`.